.travis.yml

```diff
@@ -26,7 +26,7 @@ script:
   - make -C src/ui docker-build # Tests disabled for now
 after_success:
   - |
-    if [[ "$TRAVIS_PULL_REQUEST" == false ]]; then
+    if [[ "$TRAVIS_PULL_REQUEST" = "false" ]] && [[ "$TRAVIS_BRANCH" = "master" || -n "$TRAVIS_TAG" ]]; then
```

Comment:
we only want to run this on master and tags, and not pull-requests.

Reply:
And this is because your branch was triggering the build of the images?

Reply:
It's to prevent it, it wasn't triggering any builds at all before. The last time `latest` was pushed was 23 days ago, with the last commit to master.